third_party/2and3/requests/sessions.pyi

```diff
@@ -16,6 +16,7 @@ from . import adapters
 from . import status_codes
 
 BaseAdapter = adapters.BaseAdapter
+AuthBase = auth.AuthBase
```

Comment:
This doesn't seem to exist at runtime.

Reply:
Oh, that's because you were doing that in the context of a class that also has an `auth` attribute. You can work around it by making an alias like `_auth` referring to the `auth` module.